pom.xml

```diff
@@ -42,8 +42,33 @@
     <module>jetty</module>
   </modules>
 
+  <properties>
+    <!-- Remove once parent updated -->
```

Comment:
don't know what is the accepted form here.  Do we annotated it with TODO?

Reply:
I guess i don't care as long as its reasonably prominent. If we want to adopt the TODO I guess that makes sense. LMK